.github/workflows/main.yml

```diff
@@ -15,7 +15,6 @@ jobs:
         pip install -U wheel
         pip install nbdev jupyter
         pip install .
-        pip install -r requirements.txt
```

Comment:
* Haven't checked this fully, but I believe we have similar statements in other actions too, which probably should be updated.
* I also would advise to follow the template approach, where you test both `.` and `.[stable]` to make sure both actually work.
* I thought there was als a create release action. Make sure that one has a `.[stable,gui-stable]`.

Reply:
Ah, all the release .sh/.bat files (not the github actions) also require a `.[gui]` of course.